yb-voyager/src/srcdb/postgres.go

```diff
@@ -180,3 +190,33 @@ func (pg *PostgreSQL) ExportDataPostProcessing(exportDir string, tablesProgressM
 	}
 	dfd.Save()
 }
+
+//find correct path of pg command depending on the version required
```

Comment:
// Given a PG command name ("pg_dump", "pg_restore"), find absolute path of
    // the executable file having version >= `PG_COMMAND_VERSION`.

Always put a white-space after `//` or `/*`.